website/src/content/development/testing.mdx

```diff
@@ -30,3 +32,500 @@ To test React components, we recommend to use one of the following approaches:
+              variables: { token: 'foo-bar' },
+            },
+            result: { data: { account: { balance: 300 } } },
+          },
+        ],
+      });
+      await rendered.findByText('Loading...');
+      await wait(() => {
+        expect(rendered.queryByText('Your balance is 300€')).toBeInTheDocument();
+      });
+    });
+  });
+  ```
+
+* Using SDK mocks
+
+  ```jsx
+  import { useOnActionError } from '@commercetools-frontend/actions-global';
+  import {
+    actions as sdkActions,
+    useAsyncDispatch,
+  } from '@commercetools-frontend/sdk';
+
+  const initialState = {
+    isLoading: true,
+  }
+  const reducer = (state = initialState, action) => {
+    switch (action.type) {
+      case 'success':
+        return { isLoading: false, data: action.payload };
+      case 'failure':
+        return { isLoading: false, error: action.payload };
+      default
+        return state
+    }
+  };
+
+  const BankAccountBalance = props => {
+    const [state, dispatch] = React.useReducer(reducer, initialState);
+    const dispatchFetchAction = useAsyncDispatch();
+    const onActionError = useOnActionError();
+    React.useEffect(() => {
+      try {
+        const response = await dispatchFetchAction(
+          sdkActions.get({
+            uri: '/account/balance',
+            headers: {
+              Authorization: props.token,
+            },
+          })
+        );
+        dispatch({ type: 'success', payload: response.balance });
+      } catch (error) {
+        dispatch({ type: 'failure', payload: error });
+        onActionError(error);
+      }
+    }, [props.token]);
+
+    if (state.isLoading) {
+      return (<span>'Loading..'</span>);
+    }
+    return (<span>`Your balance is ${this.state.accountBalance}€`</span>);
+  };
+
+  export default BankAccountBalance;
+  ```
+
+  ```jsx
+  import { renderAppWithRedux } from '@commercetools-frontend/application-shell/test-utils';
+  import BankAccountBalance from './bank-account-balance';
+
+  describe('BankAccountBalance', () => {
+    it('should render the balance', async () => {
+      const rendered = renderAppWithRedux(
+        <BankAccountBalance token="foo-bar" />,
+        {
+          sdkMocks: [
+            {
+              action: {
+                type: 'SDK',
+                payload: {
+                  method: 'GET',
+                  uri: '/account/balance',
+                  headers: {
+                    Authorization: 'foo-bar',
+                  },
+                },
+              },
+              response: {
+                balance: 300,
+              },
+            },
+          ],
+        }
+      );
+      await rendered.findByText('Loading...');
+      await wait(() => {
+        expect(rendered.queryByText('Your balance is 300€')).toBeInTheDocument();
+      });
+    });
+  });
+  ```
+
+* Using feature toggles
+
+  ```jsx
+  import { renderApp } from '@commercetools-frontend/application-shell/test-utils';
```

Comment:
Feel free to link to flopflips docs here again if you want to as there are other options too :)